finetuner/__init__.py

```diff
@@ -124,6 +126,7 @@ def fit(
     :param run_name: Name of the run.
     :param description: Run description.
     :param experiment_name: Name of the experiment.
+    :param model_options: Additional arguments to pass to the model construction.
```

Comment:
can we explain here what we could use? at least user can check developer reference to see what we can provide